src/main/resources/pack.mcmeta

```diff
@@ -1,6 +1,6 @@
 {
    "pack": {
-      "pack_format": 4,
+      "pack_format": 5,
       "description": "Forge resource pack"
```

Comment:
Unrelated but sure whatever

Reply:
yeah, the datapack icon was just saying out of date, so fixed it